actor/builtin/storagemarket/storagemarket.go

```diff
@@ -193,6 +198,56 @@ func (sma *Actor) UpdateStorage(vmctx exec.VMContext, delta *types.BytesAmount)
 	return 0, nil
 }
 
+func (sma *Actor) GetLateMiners(vmctx exec.VMContext) (*map[string]uint64, uint8, error) {
+	if err := vmctx.Charge(actor.DefaultGasCost); err != nil {
+		return nil, exec.ErrInsufficientGas, errors.RevertErrorWrap(err, "Insufficient gas")
+	}
+	var state State
+	ctx := context.Background()
+
+	ret, err := actor.WithState(vmctx, &state, func() (interface{}, error) {
```

Comment:
This read-only method can be simplified a bit by using the new `actor.ReadState` method instead of `actor.WithState`